pages/results.js

```diff
@@ -5,16 +5,76 @@ import Layout from "../components/Layout/Layout";
 // import calculateEmissions from "../utils/calculateEmissions";
 import capitalize from "../utils/capitalize";
 import { daysOfWeek } from "../utils/constants";
-import { getFormResponses } from "../utils/dbApi";
+import { getTripCounts } from "../utils/dbApi";
+import Highcharts from 'highcharts';
+import HighchartsExporting from 'highcharts/modules/exporting';
+import HighchartsReact from 'highcharts-react-official';
 
-export default function Results() {
+if (typeof Highcharts === 'object') {
+  HighchartsExporting(Highcharts)
+}
+
+export async function getServerSideProps() {
+  const tripjson = await getTripCounts();
```

Comment:
There's a note on the `getServerSideProps` [docs](https://nextjs.org/docs/basic-features/data-fetching#getserversideprops-server-side-rendering) (at the end of the section) that says not to use `fetch()` to call an internal API route, which `getTripCounts()` does. It doesn't explain why, but I'm guessing it might be performance related? No idea if it's worth looking into.

Reply:
Hi @ymloh !

Thx for picking that up; brilliant find! :+1: 

I've updated `getServerSideProps()` to call make the Sheets api call directly; and that seems to work fine! (the bar chart is still displaying trip count data as expected)